tests/Unit/Evolution/Systems/Cce/Test_KleinGordonWorldtubeData.cpp

```diff
@@ -0,0 +1,362 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+
+#include "Evolution/Systems/Cce/ReducedWorldtubeModeRecorder.hpp"
+#include "Evolution/Systems/Cce/WorldtubeBufferUpdater.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Helpers/Evolution/Systems/Cce/BoundaryTestHelpers.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "Utilities/Serialization/RegisterDerivedClassesWithCharm.hpp"
+
+namespace Cce::TestHelpers {
+
+// Compiler asks for previous declaration
+void create_fake_time_varying_klein_gordon_modal_data(
```

Comment:
Was it the compiler or clang-tidy? If it was clang-tidy just put `// NOLINTNEXTLINE` above the function

Reply:
This warning usually means you've forgotten an anonymous namespace, which is, in fact, the case here.